cumulusci/tasks/github/public_repo.py

```diff
@@ -0,0 +1,148 @@
+
+    def _download_repo(self, path):
+        zf = download_extract_github_from_repo(
+            self.get_repo(),
+            ref="tags/{}".format(
+                self.project_config.get_tag_for_version(self.options["version"])
+            ),
+        )
+        with cd(path):
+            zf.extractall()
+
+    def _exclude_files(self, path):
+        with cd(path):
+            for path in self.options["exclude"]:
+                if os.path.isdir(path):
+                    shutil.rmtree(path)
+                elif os.path.isfile(path):
+                    os.remove(path)
+
+    def _create_commit(self, path):
+        committer = CommitDir(self.target_repo)
+        message = f"Publishing release {self.options['version']}"
+        commit = committer(path, self.options["branch"], commit_message=message)
+        return commit.sha
+
+    def _create_release(self, path, commit):
+        # Get current release info
+        tag_name = self.project_config.get_tag_for_version(self.options["version"])
+        repo = self.get_repo()
+        # Get the ref
+        try:
+            ref = repo.ref(f"tags/{tag_name}")
+        except github3.exceptions.NotFoundError:
+            message = f"Ref not found for tag {tag_name}"
+            self.logger.error(message)
+            raise GithubException(message)
+        # Get the tag
+        try:
+            tag = repo.tag(ref.object.sha)
+        except github3.exceptions.NotFoundError:
+            message = f"Tag {tag_name} not found"
+            self.logger.error(message)
+            raise GithubException(message)
+        # Get the release
+        try:
+            release = repo.release_from_tag(tag_name)
+        except github3.exceptions.NotFoundError:
+            message = f"Release for {tag_name} not found"
+            self.logger.error(message)
+            raise GithubException(message)
+
+        # Check for tag in target repo
+        target_ref = None
+        try:
+            target_ref = self.target_repo.ref(f"tags/{tag_name}")
+        except github3.exceptions.NotFoundError:
+            pass
+        if target_ref:
+            message = "Ref for tag {tag_name} already exists in target repo"
+            raise GithubException(message)
+
+        # Create the tag
+        target_tag = self.target_repo.create_tag(
+            tag=tag_name,
+            message=tag.message,
+            sha=commit,
+            obj_type="commit",
+            tagger={
+                "name": self.github_config.username,
+                "email": self.github_config.email,
+                "date": "{}Z".format(datetime.utcnow().isoformat()),
+            },
+            lightweight=False,
+        )
+
+        # Create the release
+        body = release.body if self.options["release_body"] else ""
+        target_release = self.target_repo.create_release(
+            tag_name=tag_name,
+            name=self.options["version"],
+            prerelease=release.prerelease,
+        )
+
```

Comment:
Please add at least one logline at the info level to confirm that the release was created.

Also this blank line at the end of the file is causing the lint check to fail. You might need to double check that you have the precommit hooks installed (which should catch this before committing -- see CONTRIBUTING.rst) or reinstall requirements to make sure you have the latest version of black.